README.md

```diff
@@ -99,12 +99,15 @@ mix ns.build
 After the system builds successfully, open up another terminal window. Build
 your Nerves project in this one. There are special environment variables that
 will tell the Nerves build system to use your custom-built system rather than
-the one referenced in the `mix.exs`. To load these variables in your
-environment, run:
+the one referenced in the `mix.exs`.
 
+To load these variables in your environment, first find the `nerves.env.sh`
+script in the output directory that corresponds to your Nerves project's target.
+Then run it in the terminal for your Nerves project. For example if your Nerves
+target is Raspberry Pi Zero, the script path may look like this:
 
 ```sh
-. ~/path/to/nerves_systems/o/rpi0/nerves-env.sh
+~/path/to/nerves_systems/o/rpi0/nerves-env.sh
```

Comment:
The `.` is needed so that the environment variables exported in the `nerves-env.sh` script get included in your environment. Removing the `.` shouldn't have worked for you.